README.md

```diff
@@ -61,13 +61,13 @@ TiDB Cloud is the fully-managed service of TiDB, currently available on AWS and
 
 Quickly check out TiDB Cloud with [a free trial](https://tidbcloud.com/free-trial).
 
-See [TiDB Cloud Quick Start Guide](https://docs.pingcap.com/tidbcloud/tidb-cloud-quickstart).
+See the [TiDB Cloud Quick Start Guide](https://docs.pingcap.com/tidbcloud/tidb-cloud-quickstart).
 
 ### Start with TiDB
 
 See [TiDB Quick Start Guide](https://docs.pingcap.com/tidb/stable/quick-start-with-tidb).
 
-### Start developing TiDB
+### Start developing with TiDB
```

Comment:
we mean "contribute to tidb's codebase" rather than "use tidb to develop your own project"

Reply:
For the sentence we put in line 72, it should be "Start developing TiDB" here. Perhaps we should update the description in line 72. For contributing to TiDB's code, it's also covered in the following "[Contributing](https://github.com/pingcap/tidb/blob/master/README.md#contributing)" section.

Reply:
I changed it to this: 
```
### Contributing to TiDB development
```
I think this is the clearest way to write this.